components/script/dom/window.rs

```diff
@@ -183,6 +184,8 @@ pub struct Window {
     history: MutNullableJS<History>,
     custom_element_registry: MutNullableJS<CustomElementRegistry>,
     performance: MutNullableJS<Performance>,
+    #[ignore_heap_size_of = "Arc"]
+    paint_time_metrics: Arc<PaintTimeMetrics>,
```

Comment:
Ok, fair enough. I'll remove it from Window until we implement the PerformanceObserver API then.